src/output-json-flow.c

```diff
@@ -273,6 +273,12 @@ static void EveFlowLogJSON(OutputJsonThreadCtx *aft, JsonBuilder *jb, Flow *f)
     if (f->flags & FLOW_WRONG_THREAD)
         JB_SET_TRUE(jb, "wrong_thread");
 
+    if (f->flags & FLOW_ACTION_DROP) {
```

Comment:
@jasonish any thoughts on how to add this to the flow records?

Reply:
This seems fine.  The alert records use "blocked" and seem to take into consideration if the engine is running in IPS mode. While I prefer the naming here, it would be nice to be consistent given the field name is also "action" in the alert records.